spec/rails_helper.rb

```diff
@@ -41,5 +41,15 @@
         'Host'=>'www.thecocktaildb.com'
         }).
       to_return(status: 200, body: nil, headers: {})
+
+    stub_request(:get, "https://www.thecocktaildb.com/api/json/v1/1/lookup.php?i=16158").
+        with(
+          headers: {
+       	  'Accept'=>'*/*',
+       	  'Accept-Encoding'=>'gzip;q=1.0,deflate;q=0.6,identity;q=0.3',
+       	  'Host'=>'www.thecocktaildb.com'
+          }).
```

Comment:
make sure that you keep the formatting consistent between file for whether you use a 2 space tab or a 4 space tab